lua/net.lua

```diff
@@ -90,6 +91,13 @@ return function (gatekeeper_server)
 		end
 	end
 
+	-- Setup the user and file permissions that
+	-- Gatekeeper runs on after it boots.
+	ret = staticlib.c.gatekeeper_setup_user(net_conf, user)
```

Comment:
1. The comment is wrong since file permissions are not affected by `staticlib.c.gatekeeper_setup_user()`.
2. Only call `staticlib.c.gatekeeper_setup_user()` when `user` is not `nil`.